### PR TITLE
Update data.json to include MTA on BASE

### PR DESCRIPTION
**Pull request by taminater**
**Description**

Adding contract address for MTA on Base

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by taminater** [comment]
@roberto-bayardo

File: data/MTA/data.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     },
     "optimism": {
       "address": "0x929b939f8524c3be977af57a4a0ad3fb1e374b50"
+    },
+    "base":     {
+      "address": "0x9cc2fc2f75768b0307925c7935396ec9d94bba44"
     }
-  }
```
Comment (roberto-bayardo) [inline data/MTA/data.json:14]: need to restore this } so the json will parse.

**Comment by roberto-bayardo** [comment]
Could you recreate the L2 token using the alternate factory now listed in our docs & on the readme?  The predeploy generates addresses that conflict with other tokens on optimism.

https://docs.base.org/base-contracts/#l2-contract-addresses

**Comment by roberto-bayardo** [comment]
Superceded by https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/521

**Pull request closed without merging**